testnet/run_libp2p.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/sh
+
+set -e
+
+NET_NODE_KEY=$(./etc/nomos/etcd/register_etcd.sh)
+CONSENSUS_NODE_KEY=$NET_NODE_KEY
+
+node_ids=$(etcdctl get "/node/" --prefix --keys-only)
+for node_id in $node_ids; do
+	node_key=$(etcdctl get "/config${node_id}/key" --print-value-only)
+	node_ip=$(etcdctl get "/config${node_id}/ip" --print-value-only)
+	node_multiaddr="/ip4/${node_ip}/tcp/3000"
+
+	if [ -z "$OVERLAY_NODES" ]; then
+		OVERLAY_NODES=$node_key
+		NET_INITIAL_PEERS=$node_multiaddr
+	else
+		OVERLAY_NODES="${OVERLAY_NODES},${node_key}"
+		NET_INITIAL_PEERS="${NET_INITIAL_PEERS},${node_multiaddr}"
+	fi
+done
+
+echo "I am a libp2p container ${HOSTNAME} node ${NET_NODE_KEY}"
```

Comment:
Do we really need two different files - `testnet/run_mixnet.sh` and `testnet/run_libp2p.sh` - that do the same thing, with the only difference being this `echo`?

Reply:
Removed bootstrap and mixnet placeholders. We'll add those in a separate PR.